src/ranking.jl

```diff
@@ -171,3 +171,108 @@ Missing values are assigned rank `missing`.
 """
 tiedrank(x::AbstractArray; sortkwargs...) =
     _rank(_tiedrank!, x, Float64; sortkwargs...)
+
+"""
+    quantilerank(a::RealVector, score)
+    quantilerank(a::RealVector, score::RealVector)
+
+Return the percentile-position of score (0-1) relative to a.
+
+For example, a quantilerank of 80% means that 80% of the values in `a` are below the given score.
+
+# Arguments
+* `a`: Array of values to which `score` is compared.
+* `score`: Value that is compared to the elements in `a`.
+* `method` (optional, default `:rank`):  method used for calculate the quantilerank. Options available (`:rank`, `:weak`, `:strict`, `:mean`).
```

Comment:
Yes, I want to avoid using scipy's docstring. I really liked the `quantile` docstring from `Statistics.jl` and it will be my inspiration and reference for the new docstring.

Once we've defined the next steps on the issues presented, I'll add new examples and write a new docstring to the `quantile` templates